lib/src/test/java/com/auth0/jwt/JWTVerifierTest.java

```diff
@@ -587,7 +599,20 @@ public void shouldRemoveClaimWhenPassingNull() throws Exception {
                 .build();
 
         assertThat(verifier.claims, is(notNullValue()));
-        assertThat(verifier.claims, not(hasKey("iss")));
+        assertThat(verifier.claims, not(hasKey("sub")));
```

Comment:
This should remain the same. When we pass `null` we're removing that claim check.

```suggestion
        assertThat(verifier.claims, not(hasKey("iss")));
```